### ISSUE TITLE
kendoMultiSelectSummaryTag doesn't work with Angular compiler option strictTemplates

### ISSUE THREAD

**Issue by IT-CASADO** [telerik/kendo-angular#3158]
**Describe the bug**
If you enable 'strictTemplates' compiler option you get an error:

my.component.html:9:7 - error TS2322: Type 'string' is not assignable to type 'number'.

                                       kendoMultiSelectSummaryTag

**To Reproduce**
```
        <kendo-multiselect
          kendoMultiSelectSummaryTag
          ....
        >
        </kendo-multiselect>
```

tsconfig.json

```
{
        ....
	"angularCompilerOptions": {
		"strictTemplates": true,
		"strictInjectionParameters": true,
		"strictInputAccessModifiers": true,
		"strictDomEventTypes": false
	}
        ....
}
```

[You can use one of your examples!](https://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/summary-tag-mode/)

**Expected behavior**
Success on compiling with IVY and strictTemplates enabled!

**Comment by IT-CASADO** [comment]
I'm also having issues with kendo-popup and strictTemplates enabled: #2865

**Comment by IT-CASADO** [comment]
[Angular provide us with a solution for this problem](https://angular.io/guide/structural-directives#improving-template-type-checking-for-custom-directives).

**Comment by dimitar-pechev** [comment]
Thank you for reporting this problem, we'll handle it with higher priority. 

The problem is that the directive selector is also an input property of type `number`. You can work-around the problem for now by specifying explicitly the maximum number of standalone tags before rendering a summary tag. [Here's an example](https://stackblitz.com/edit/angular-gydooa?file=app/app.component.ts). If you want all selected tags to be combined in a single summary tag, provide `0` to the directive selector:

```
[kendoMultiSelectSummaryTag]="0"
```

**Comment by IT-CASADO** [comment]
I can confirm that your suggestion is working.

As I mentioned before #2865 is also a problem for us. It's falling into the same category (strict compiling) and I see that you are participating on the issue as a member.

Many thanks so far.

**Comment by svetq** [comment]
The fix is available in the official v`5.2.0` of the @progress/kendo-angular-dropdowns package.
